spring-data-multi-storage-transaction-sample/database.properties

```diff
@@ -0,0 +1,12 @@
+scalar.db.storage=multi-storage
```

Comment:
Is this `database.properties` file used? It seems only `database-docker.properties` is required to init the docker environment.

Reply:
Good catch! Thanks.

Done https://github.com/scalar-labs/scalardb-samples/pull/22/commits/ffd7e876d9313754311f734f8837fe9b82db29c2

Reply:
Good catch! Thanks.

Done https://github.com/scalar-labs/scalardb-samples/pull/22/commits/ffd7e876d9313754311f734f8837fe9b82db29c2